TELCODOCS-1731 - Update low latency tuning docs (#71797)
openshift/openshift-docs: modules/cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class.adoc

// Module included in the following assemblies:
//
// * scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc

:_mod-docs-content-type: PROCEDURE
[id="cnf-node-tuning-operator-creating-pod-with-guaranteed-qos-class_{context}"]

> = Creating a pod with a QoS class of `Guaranteed`

Reviewer: Doc Guidelines: [Do not use backticks or other markup in assembly or module headings.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings)

```suggestion
= Creating a pod with the Guranteed QoS class of Guaranteed
```
Or 
```suggestion
= Creating a pod with a Guaranteed QoS class
```

* Every container in the pod must have a CPU limit and a CPU request, and they must be the same.

The following example shows the configuration file for a pod that has one container. The container has a memory limit and a memory request, both equal to 200 MiB. The container has a CPU limit and a CPU request, both equal to 1 CPU.

[source,yaml]
----
apiVersion: v1
kind: Pod
metadata:
  name: qos-demo
  namespace: qos-example
spec:
  securityContext:
    runAsNonRoot: true
    seccompProfile:
      type: RuntimeDefault
  containers:
  - name: qos-demo-ctr
    image: <image-pull-spec>
    resources:
      limits:
        memory: "200Mi"
        cpu: "1"
      requests:
        memory: "200Mi"
        cpu: "1"
    securityContext:
      allowPrivilegeEscalation: false
      capabilities:
        drop: [ALL]
----

. Create the pod:
+
[source,terminal]
----
$ oc  apply -f qos-pod.yaml --namespace=qos-example
----

. View detailed information about the pod:
+
[source,terminal]
----
$ oc get pod qos-demo --namespace=qos-example --output=yaml
----
+
.Example output
[source,yaml]
----
spec:
  containers:
    ...
status:
  qosClass: Guaranteed
----
+
[NOTE]
====

> If a container specifies its own memory limit, but does not specify a memory request, {product-title} automatically assigns a memory request that matches the limit. Similarly, if a container specifies its own CPU limit, but does not specify a CPU request, {product-title} automatically assigns a CPU request that matches the limit.

Reviewer: _specifies its own_  is a bit awkward to me. Is it even accurate?

```suggestion
If you specify a memory limit for a container, but do not specify a memory request, {product-title} automatically assigns a memory request that matches the limit. Similarly, you specify a  CPU limit for a container, but do not specify a CPU request, {product-title} automatically assigns a CPU request that matches the limit.
```
